packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/lambda/call-aws-service-cross-region.ts

```diff
@@ -0,0 +1,184 @@
+import { Construct } from 'constructs';
+import * as iam from '../../../aws-iam';
+import { IFunction } from '../../../aws-lambda';
+import * as sfn from '../../../aws-stepfunctions';
+import { Token, Duration } from '../../../core';
+import { CrossRegionAwsSdkSingletonFunction } from '../../../custom-resource-handlers/dist/aws-stepfunctions-tasks/cross-region-aws-sdk-provider.generated';
+
+/**
+ * Properties for calling an AWS service's API action from your
+ * state machine across regions.
+ */
+export interface CallAwsServiceCrossRegionProps extends sfn.TaskStateBaseProps {
```

Comment:
This seems to be re-defining a lot of props from `CallAwsServiceProps`. Any reason we're not extending `CallAwsServiceProps` instead?

Reply:
They look similar but are slightly different, e.g. `service` property's name format. Also I'd like to completely decouple them to avoid unwanted side effect to the stable CallAwsService construct.

Reply:
Fair point, decoupling makes sense to me